recipes/grpc-proto/all/conanfile.py

```diff
@@ -45,25 +47,23 @@ def configure(self):
 
     def validate(self):
         if self.settings.compiler.cppstd:
-            tools.check_min_cppstd(self, 11)
+            check_min_cppstd(self, 11)
 
         if self.options.shared and (not self.options["protobuf"].shared or not self.options["googleapis"].shared):
             raise ConanInvalidConfiguration("If built as shared, protobuf and googleapis must be shared as well. Please, use `protobuf:shared=True` and `googleapis:shared=True`")
 
+    def generate(self):
+        tc = CMakeToolchain(self)
+        tc.cache_variables["GOOGLEAPIS_PROTO_DIRS"] = self.dependencies["googleapis"].cpp_info.resdirs[0].replace("\\", "/")
```

Comment:
```suggestion
        tc.cache_variables["GOOGLEAPIS_PROTO_DIRS"] = self.dependencies["googleapis"].cpp_info.aggregated_components().resdirs[0].replace("\\", "/")
```

see https://github.com/conan-io/conan-center-index/issues/15599